website/docs/artist_hosts_maya_xgen.md

```diff
@@ -4,26 +4,94 @@ title: Xgen for Maya
 sidebar_label: Xgen
 ---
 
-## Working with Xgen in OpenPype
+## Setup
 
-OpenPype support publishing and loading of Xgen interactive grooms. You can publish 
-them as mayaAscii files with scalps that can be loaded into another maya scene, or as
-alembic caches. 
```

Comment:
The new documentation gets technical rather quickly - should we maybe, like before have a few lines of 'introduction' of _what_ the Xgen functionality in OpenPype is intended to do. Not something long, but at least an artist looking at "hey what's supported" could find it in a quick read instead of first having to read about how they need to customize project settings.

Reply:
Sure, will add a little intro.